config/routes.rb

```diff
@@ -18,16 +18,18 @@
 
   # LetsEncrypt
   get '/.well-known/acme-challenge/:id' => 'pages#letsencrypt'
-
   get '/veterans/thanks', to: 'veterans#thanks'
-  get '/veterans/map', to: 'veterans#map'
   get '/.well-known/acme-challenge/:id' => 'pages#letsencrypt'
   resources :veterans, only: [:new, :create]
   post '/veterans/claim/:veteran', to: 'veterans#claim', as: :veterans_claim
   post '/veterans/unclaim/:veteran', to: 'veterans#unclaim', as: :veterans_unclaim
 
   resources :donations, only: [:index, :new, :create]
 
+  # Maps
+  get '/maps/veterans', to: 'veterans#map'
```

Comment:
If we're updating the URL does it make sense to create a new `MapsController`? It seems like a lot of the code is similar so maybe it would clean up some duplication as well.